dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/CompanyQueryManager.kt

```diff
@@ -43,24 +43,18 @@ class CompanyQueryManager(
     fun searchCompaniesAndGetApiModel(
         filter: StoredCompanySearchFilter,
     ): List<BasicCompanyInformation> {
-        if (filter.dataTypeFilter.size == DataTypesExtractor().getAllDataTypes().size) {
-            filter.dataTypeFilter = emptyList()
-        }
-
-        return if (
+        // todo check if needed, maybe put ChunkManager functions here
```

Comment:
Die Funktion hier sollten wir kurz diskutieren.

Reply:
Die Funktionalität aus dem chunkmanager sollte extrahiert werden und dann hier und im chunkmanager aufgerufen werden. Es ist ja in beiden Fällen die Frage "Sind Filter gesetzt?" und würde sonst ein Duplikat darstellen.